.github/workflows/abm_benchmark_test.yml

```diff
@@ -0,0 +1,30 @@
+# experimental pipeline to test benchmarks on github runners
```

Comment:
Should experimental be here if we merge this?

Reply:
I think so. this is only intended to test the github runners and not meant to be used by developers for real performance comparisons. When we add the benchmark to the CI for good, it will be different, probably integrated in the main CI pipeline.

Reply:
Sadly, github does not run pipelines on a schedule unless the pipeline is on the main branch